.github/workflows/tests_linux_x86.yml

```diff
@@ -20,6 +20,8 @@ jobs:
   cpptests:
     runs-on:
       - self-hosted
+      - linux
+      - x64
```

Comment:
Thanks @rashidnhm ! Do we need to make some changes for the wheels building workflow as well?

Reply:
Should not be necessary as those workflows use GitHub hosted runners